x-pack/plugins/security_solution/common/endpoint/schema/actions.ts

```diff
@@ -69,6 +69,20 @@ export const ActionDetailsRequestSchema = {
   }),
 };
 
+const commandsSchema = schema.oneOf([
+  schema.literal('isolate'),
+  schema.literal('unisolate'),
+  schema.literal('kill-process'),
+  schema.literal('suspend-process'),
+  schema.literal('running-processes'),
+]);
```

Comment:
Question: for the commands and the Statuses (below), should we build the schema from the `constants` you created so that they remain in sync? Like:

```suggestion
const commandsSchema = schema.oneOf(
  RESPONSE_ACTION_COMMANDS.map((commandName) => schema.literary(commandName)
);
```

Reply:
Oh yes I tried that exact thing that's why I moved the action commands const, but TS doesn't like it 😢

Reply:
huh! I tried it again and seems like TS is okay with it. Perhaps VSCode was acting up earlier. 🤷

Reply:
Okay spoke too soon. So VScode was slow now and not earlier. 😅  I tried:

```
const statusesSchema = schema.oneOf(RESPONSE_ACTION_STATUS.map((status) => schema.literal(status)));
```
and TS throws me this error

```
x-pack/plugins/security_solution/common/endpoint/schema/actions.ts:81:37 - error TS2769: No overload matches this call.
 proc [tsc]   The last overload gave the following error.
 proc [tsc]     Argument of type '(Type<"failed"> | Type<"pending"> | Type<"successful">)[]' is not assignable to parameter of type '[Type<"failed" | "pending" | "successful">]'.
 proc [tsc]       Target requires 1 element(s) but source may have fewer.
```

Oddly, the schema change works fine, the API works well and the tests pass. 

Maybe I should just add a ts ignore for now?

Reply:
okay so I ignored the errors for now 89a35e29ff0c7d6b008d39a1260e9825c067cb50